src/options.rs

```diff
@@ -515,6 +515,9 @@ where
             Arg::new("vtable-generation")
                 .long("vtable-generation")
                 .help("Enables generation of vtable functions."),
+            Arg::new("sort-semantically")
+                .long("sort-semantically")
+                .help("Enables sorting of code generation in a predefined manner"),
```

Comment:
Maybe the message should suggest more strongly that the sorting is not going to *alphabetize* things, just group them. Perhaps like this:
```suggestion
                .help("Emits generated code in groups sorted in a predefined manner (for example, `struct`s before `enum`s)."),
```

This might be overkill, though; maybe the flag name `sort-semantically` is enough.